JS

```diff
@@ -0,0 +1,211 @@
+  this.calories = this[type].calories + this[stuffing].calories;
+}
+
+Hamburger.prototype.LARGE_SIZE= { 
+  name:"Big hamburger",
+  price: 100,
+  calories: 40
+};
+
+Hamburger.prototype.SMALL_SIZE= {
+  name:"Small hamburger",
+  price: 50,
+  calories: 20
+  };
+
+Hamburger.prototype.STUFFING_CHEESE = {
+  name: 'Cheese',
+  price: 10,
+  calories: 20
+};
+
+Hamburger.prototype.STUFFING_SALAD = {
+  name: 'Salad',
+  price: 20,
+  calories: 5
+};
+
+Hamburger.prototype.STUFFING_POTATO = {
+  name: 'Potato',
+  price: 15,
+  calories: 10
+};
+
+Hamburger.prototype.calculatePrice = function(){
+  return this.price*this.count;
+}
+
+Hamburger.prototype.calculateCalories = function(){
+  return this.calories*this.count;
+}
+
+Hamburger.prototype.getName = function(){
+  return this.name +' '+ this.stuffingName;
+}
+
+Hamburger.prototype.getCount = function(){
+  return this.count;
+}
+
+function SaladAndDrink(type, count){
+  this.name = this[type].name;
+  this.count = count;
+  this.price = this[type].price;
+  this.calories = this[type].calories;
+}
+
+SaladAndDrink.prototype.calculatePrice = function(){
+  return this.price*this.count;
+}
+
+SaladAndDrink.prototype.calculateCalories = function(){
+  return this.calories*this.count;
+}
+
+SaladAndDrink.prototype.getName = function(){
+  return this.name;
+}
+
+SaladAndDrink.prototype.getCount = function(){
+  return this.count;
+}
+
+SaladAndDrink.prototype.SALAD_CAESAR= { 
+  name:"Salad Caesar",
+  price: 1,
+  calories: 0.2
+};
+
+SaladAndDrink.prototype.RUSSIAN_SALAD= { 
+  name:"Russian salad",
+  price: 0.5,
+  calories: 0.8
+};
+
+SaladAndDrink.prototype.DRINK_COLA = {
+  name: 'Cola',
+  price: 50,
+  calories: 40
+};
+
+SaladAndDrink.prototype.DRINK_COFFEE = {
+  name: 'Coffee',
+  price: 80,
+  calories: 20
+};
+
+       //// different functions for a big and a small because it is more conviniently 
+let arrOrder = [];
+   
+function addBigHamburger(){
+  let count = document.getElementById('bigHamburgerCount').value;
+  let stuffing = document.getElementById('bigHamburgeStuffing').value;
+  let order = new Hamburger('LARGE_SIZE', count, stuffing);
+  if(count!=0) addCurrent(order);
+  if(!count) alert('Please enter the number');
+     //// fill the table
+  orderList();
+   /// count totall summ and calories
+   totall();
+}
+    
+function addSmallHamburger(){
+  let count = document.getElementById('smallHamburgerCount').value;
+  let stuffing = document.getElementById('smallHamburgerStuffing').value;
+  let order = new Hamburger('SMALL_SIZE', count, stuffing);
+  if(count !=0) addCurrent(order);
+  if(!count) alert('Please enter the number')
+  orderList();
+  totall();      
+}
+
+function addSalad(){
+  let type  = document.getElementById('chooseSalad').value;
+  let count = document.getElementById('saladCount').value;
+  let order = new SaladAndDrink(type, count);
+  if(count !=0) addCurrent(order);
+  if(!count) alert('Please enter the number')
+  orderList();
+  totall();
```

Comment:
@Mikleprog I would recommend to render orderList only in case if count != 0 to prevent useless DOM operations